### DISCUSSION TITLE
AP Fall Back mode

### DISCUSSION THREAD

**Discussion by surajchauhan6631** [forkineye/ESPixelStick#770] [General]
I just want to confirm what is use of AP Fall back mode. My ESPIxelstick will run remote file stored on sd card. And because there will be no access to wifi router where espixelstick will be stored. so i can't connect to esp32 through my network wifi. For that reason i have to connect directly to espixelstick wifi. For that reason what should i choose, AP FallBACK, reboon on fail, and what is stay in AP Mode.

**Comment by MartinMueller2003** [comment]
You want to set AP fallback to yes. This causes the ESP to go into AP mode so you can connect to it.
You need to set Stay in AP mode so that the ESP does not cycle between AP mode and trying to connect to a WiFi Router. Reboot on fail means: If the esp could not connect to a WiFi router AND no one connected to the ESP in AP mode then reboot the ESP. You want to turn that off.

**Reply by surajchauhan6631** [reply]
Ok, and is there anyway we can protect the espixelstick from unauthorised access, so that I can only connect to espixelstick WiFi

**Reply by surajchauhan6631** [reply]
And in ap mode what ip address I have to use to connect with espixelstick

**Reply by MartinMueller2003** [reply]
You can configure the passphrase ro that only you can attach to the ESP.

**Reply by surajchauhan6631** [reply]
Wifi connection with espixelstick in ap mode keep getting disconnected while i am setting the output 1 to dmx output and secondary input to fpp remote As soon as i click on save changes button and again  i have to choose dmx to output 1. If i set set output 1 to dmx and choose secondary input to effects and click on save changes button every works fine.

**Reply by MartinMueller2003** [reply]
Please include the console log. Not much I can do without it.

**Reply by surajchauhan6631** [reply]
Is your latest ci build is stable ?

**Reply by MartinMueller2003** [reply]
Yes my latest CI build is stable.
Also, you can use the "experimental" build at espixelstickwebflasher.from-ct.com:5000
Or the latest CI build from the Forkineye. He integrated my changes a few days ago.
